add outline of notebook

zingale opened the pull request.

zingale commented:
I believe this is ready for review for Monday.

Two notes: 

* `yt.load_sample()` did not work for me.

* While breaking the multiaxis slice plot up into cells to allow me to write descriptive text inline, I needed to add a `p.zoom(2)` call before assigning the plots to their respective axes.  I am not sure why this is the case, but somewhere the state gets lost if you divide the cells.

munkm commented:
I've submitted a PR to fix `load_sample` in https://github.com/yt-project/yt/pull/2496

zingale commented:
cool.  I stared at that exact line in the code yesterday thinking it didn't
seem right, but it didn't occur to me to add `return` :)

On Sun, Mar 22, 2020 at 1:36 PM Madicken Munk <notifications@github.com>
wrote:

> I've submitted a PR to fix load_sample in yt-project/yt#2496
> <https://github.com/yt-project/yt/pull/2496>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/yt-project/2020-03-cca-workshop/pull/2#issuecomment-602244626>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB3USJJXXDYGRL47YAZBPGLRIZD75ANCNFSM4LOWEK7A>
> .
>


-- 
Michael Zingale
Associate Professor

Dept. of Physics & Astronomy • Stony Brook University • Stony Brook, NY
11794-3800
*phone*:  631-632-8225
*e-mail*: Michael.Zingale@stonybrook.edu
*web*: h <http://www.astro.sunysb.edu/mzingale>ttp://zingale.github.io
github: http://github.com/zingale

munkm commented:
Well, somehow it didn't occur to those of us that wrote or reviewed the PR too! :-) 

I also definitely verified that it works this time. 

``` 
In [1]: import yt

In [2]: ds = yt.load_sample("IsolatedGalaxy")
yt : [INFO     ] 2020-03-22 12:31:14,779 Files located at /Users/madicken/yt_data/IsolatedGalaxy.tar.gz.untar/IsolatedGalaxy/galaxy0030/galaxy0030
yt : [INFO     ] 2020-03-22 12:31:14,779 Default to loading galaxy0030/galaxy0030 for IsolatedGalaxy dataset
yt : [INFO     ] 2020-03-22 12:31:14,826 Parameters: current_time              = 0.0060000200028298
yt : [INFO     ] 2020-03-22 12:31:14,826 Parameters: domain_dimensions         = [32 32 32]
yt : [INFO     ] 2020-03-22 12:31:14,826 Parameters: domain_left_edge          = [0. 0. 0.]
yt : [INFO     ] 2020-03-22 12:31:14,826 Parameters: domain_right_edge         = [1. 1. 1.]
yt : [INFO     ] 2020-03-22 12:31:14,826 Parameters: cosmological_simulation   = 0.0

In [3]: slc = yt.SlicePlot(ds, 'z', 'density')
Parsing Hierarchy : 100%|██████████████████████████████████████████████| 173/173 [00:00<00:00, 2718.67it/s]
yt : [INFO     ] 2020-03-22 12:31:18,367 Gathering a field list (this may take a moment.)
yt : [INFO     ] 2020-03-22 12:31:21,128 xlim = 0.000000 1.000000
yt : [INFO     ] 2020-03-22 12:31:21,128 ylim = 0.000000 1.000000
yt : [INFO     ] 2020-03-22 12:31:21,130 xlim = 0.000000 1.000000
yt : [INFO     ] 2020-03-22 12:31:21,130 ylim = 0.000000 1.000000
yt : [INFO     ] 2020-03-22 12:31:21,131 Making a fixed resolution buffer of (('gas', 'density')) 800 by 800

In [4]: exit()
```

zingale commented:
okay, I switched to `load_sample()`.  If it is not merged by tomorrow, I'll just hack my local copy of yt.

Also I added a colormap example, since it's pretty

zingale commented:
Thanks!

yes, I was going to verbally mention the imagemagick bit.  

I thought I had an explanation of native.  I'll make sure that is emphasized.

zingale commented:
I have this for native:


----

There are a few things to note here:

* This is an example of an axis-aligned slice plot—we are using the natural coordinate axes of the dataset to define the slice normal.
* The coordinates have been shifted from their true values to have (0, 0) at the center. This can be a big source of confusion, so we'll undo that below.
* The units appear on the axes and data range
* A log scale has been automatically applied to the data range. This is true for most fields.

We can fix the coordinate problem by passing origin = "native" to SlicePlot:

munkm commented:
You totally do and I missed it. Sorry!
